config/dev.webpack.config.js

```diff
@@ -12,23 +16,22 @@ const insightsProxy = {
 const webpackProxy = {
   deployment: process.env.BETA ? 'beta/apps' : 'apps',
   appUrl: process.env.BETA ? ['/beta/insights/advisor'] : ['/insights/advisor'],
-  env: `prod-stable`, // pick chrome env ['ci-beta', 'ci-stable', 'qa-beta', 'qa-stable', 'prod-beta', 'prod-stable']
+  env: process.env.CHROME_ENV ? process.env.CHROME_ENV : 'stage-stable', // pick chrome env ['stage-beta', 'stage-stable', 'prod-beta', 'prod-stable']
   useProxy: true,
   proxyVerbose: true,
-  useCloud: false,
   routes: {},
   customProxy: [
-    {
-      context: (path) => path.includes('/api/'),
-      target: 'https://console.redhat.com/',
-      secure: true,
-      changeOrigin: true,
-      autoRewrite: true,
-      ws: true,
-      onProxyReq: function (request) {
-        request.setHeader('origin', 'https://console.redhat.com/');
-      },
-    },
+    // {
```

Comment:
I'm not sure what this was meant for, but the frontend works without it and breaks (showing "No access") with it.
If it was meant to be used for something and someone knows, it is still here.